src/WebformBuilder.js

```diff
@@ -1237,6 +1237,7 @@ export default class WebformBuilder extends Component {
           _.pull(newComp.validators, 'required');
           parentComponent.tabs[tabIndex].splice(index, 1, newComp);
           newComp.checkValidity = () => true;
+          newComp.checkComponentValidity = () => true;
```

Comment:
Why do not we just add the required day property to the properties list that must be omitted for the default value field (line 1200)?

Reply:
updated